### ISSUE TITLE
Provide packages for latest openSUSE Leap releases

### ISSUE THREAD

**Issue by sledz** [git-lfs/git-lfs#3522]
Please provide packages at PackageCloud for latest openSUSE Leap releases (15.0 and 15.1). The last provided is for outdated 42.3.

Thx

**Comment by bk2204** [comment]
Hey, I think @andyneff added support for this in #3515. These should be included in our next release, which is tentatively planned for Friday.

**Comment by sledz** [comment]
#3515 is for SLES 15 not for openSUSE Leap 15.0 or 15.1.

@andyneff, or does #3515 cover them all?

**Comment by bk2204** [comment]
I think you're right that this is limited to SLES. If you use the packages for SLES, they will likely work, but I'll reopen this to get packages for OpenSUSE specifically. It's likely that at this point they won't be available until 2.8.0, since I'm about to start on the 2.7.0 release in a few moments.

**Comment by sledz** [comment]
Hmmm?

@andyneff, running `curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh | sudo bash` under openSUSE 15.0 generates these two additional repo entries:

```
#  | Alias                           | Name                                    | Enabled | GPG Check | Refresh | Priority | Type   | URI                                                                      | Service
---+---------------------------------+-----------------------------------------+---------+-----------+---------+----------+--------+--------------------------------------------------------------------------+--------
 ...
 7 | github_git-lfs                  | github_git-lfs                          | Yes     | (r ) Yes  | Yes     |   99     | rpm-md | https://packagecloud.io/github/git-lfs/opensuse/15.0/x86_64              |        
 8 | github_git-lfs-source           | github_git-lfs-source                   | Yes     | (r ) Yes  | Yes     |   99     | rpm-md | https://packagecloud.io/github/git-lfs/opensuse/15.0/SRPMS               |    
```
But there are no packages available in these locataions. :(

So this issue is not yet done and should be reopened!

**Comment by andyneff** [comment]
@sledz You won't see any until we have a release after the PR was accepted. It looks like we just missed the 2.7.1, but it is on the [next](https://github.com/git-lfs/git-lfs/milestone/24?closed=1) milestone

**Comment by sledz** [comment]
Is there a forecast for this release?

**Comment by bk2204** [comment]
We may or may not do a v2.7.2. If we do, it'll be included then; otherwise, it will be included in 2.8.0, which will likely be around June or so.
